tests/testOperations.js

```diff
@@ -23,7 +23,7 @@ function TestOperations(page) {
 
 	this.assertFocussedElementId = function (expectedId) {
 		page.getFocussedElementId().then(function (id) {
-			assert.notEqual(-1, id.indexOf(expectedId), 'The focused element did not have the expected id ' + expectedId);
+			assert.notEqual(id.indexOf(expectedId), -1, 'The focused element did not have the expected id ' + expectedId);
```

Comment:
it could :) ... there is a whole host of improves to the test suite that can be done, any help is *more* than appreciated